x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/user_added_info.test.tsx

```diff
@@ -20,9 +18,103 @@ describe('UserAddedInfo', () => {
     roleType: 'user',
   };
 
-  it('renders', () => {
+  it('renders with email', () => {
     const wrapper = shallow(<UserAddedInfo {...props} />);
 
-    expect(wrapper.find(EuiText)).toHaveLength(6);
+    expect(wrapper).toMatchInlineSnapshot(`
+      <Fragment>
+        <EuiText
+          size="s"
+        >
+          <strong>
+            Username
+          </strong>
+        </EuiText>
+        <EuiText
+          size="s"
+        >
+          user1
+        </EuiText>
+        <EuiSpacer />
+        <EuiText
+          size="s"
+        >
+          <strong>
+            Email
+          </strong>
+        </EuiText>
+        <EuiText
+          size="s"
+        >
+          test@test.com
+        </EuiText>
+        <EuiSpacer />
+        <EuiText
+          size="s"
+        >
+          <strong>
+            Role
+          </strong>
+        </EuiText>
+        <EuiText
+          size="s"
+        >
+          user
+        </EuiText>
+        <EuiSpacer />
+      </Fragment>
+    `);
+  });
+
+  it('renders without email', () => {
+    const wrapper = shallow(<UserAddedInfo {...props} email="" />);
+
+    expect(wrapper).toMatchInlineSnapshot(`
```

Comment:
nice, glad you got to use inline snapshots so soon! test names are great as well for helping understand/look for the rendered diffs 💯